src/node_crypto.cc

```diff
@@ -3811,20 +4568,20 @@ void Verify::VerifyFinal(const FunctionCallbackInfo<Value>& args) {
   Verify* verify;
   ASSIGN_OR_RETURN_UNWRAP(&verify, args.Holder());
 
-  char* kbuf = Buffer::Data(args[0]);
-  ssize_t klen = Buffer::Length(args[0]);
+  unsigned int offset = 0;
+  ManagedEVPPKey pkey = GetPublicKeyFromJS(args, &offset, true, true);
```

Comment:
How about something like
```suggestion
  ManagedEVPPKey pkey = GetPublicKeyFromJS(args, &offset, true, true);
  if (nullptr != pkey) offset++;
```

Reply:
I don't think I understand what your suggested change is supposed to do.

Reply:
I dismissed it. But my intention way to omit the `&offset` in-out arg, and decide the initial value of offset based on the value of `pKey`. But I looked at you recent implementation, and I'm not sure that is easy.